python_cross_site_scripting/2239923138_Trash-Bin/ai.py

```
@@ -0,0 +1,60 @@
+def index():
+    query = None
+    answer = None
+    if request.method == 'POST':
+        query = request.form.get('query')  # 获取前端输入的问题
+        if query:
+            answer = get_answer(query)  # 调用回答逻辑生成回应
+    # 渲染模板，把问题和回答传给前端页面
+    return render_template('ai_.html', query=query, answer=answer)
+
+if __name__ == '__main__':
+    app.run(debug=True)  # 调试模式，方便开发时看报错，生产环境需调整
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)